extras/mmu.py

```diff
@@ -140,8 +140,8 @@ class Mmu:
     ENDSTOP_EXTRUDER_TOUCH     = "mmu_ext_touch"
     ENDSTOP_GEAR_TOUCH         = "mmu_gear_touch"
     ENDSTOP_GATE               = "mmu_gate"
-    ENDSTOP_EXTRUDER           = "mmu_extruder"
-    ENDSTOP_TOOLHEAD           = "mmu_toolhead"
+    ENDSTOP_EXTRUDER           = "extruder"
+    ENDSTOP_TOOLHEAD           = "toolhead"
```

Comment:
Maybe it isn't desirable to change these names; please comment if I should change it back to the `mmu_*` format instead.  I switched to these names because that seems to be what e.g. the `[mmu_sensors]` seems to generate, and also the names that the old config format used to use.

Reply:
Not quite sure what isn't working, but the convention I used is that all endstops for Happy Hare are:
`mmu_XXX`

The filament sensor rendering is really for cosmetics. They are named `mmu_XXX` for sensors that are part of the MMU, and just `XXX` if they are part of the printer.    The "extruder" and "toolhead" have always been named without the mmu because they started life as standalone sensors.